mbed-edge-core-devmode/deb/debian/launch-edge-core.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020, Arm Limited and affiliates.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------
+
```

Comment:
Do we need to check for existence of the mcc_config folder here, similar to [mbed-edge-core's version of this script](https://github.com/PelionIoT/distro-pelion-edge/blob/a9f2cc0ad079991ec8570100263548ffe58cde11/mbed-edge-core/deb/debian/launch-edge-core.sh#L20)?

Reply:
For dev-mode, I believe that the mcc_config folder will be created by edge-core whereas in production mode, the mcc_config folder has to be created by the FCC before edge-core starts up.

I don't think the mcc_config folder will ever exist before mbed-edge starts up in devmode

Reply:
ok understood